dist.ts

```diff
@@ -0,0 +1,196 @@
+import { Temporal } from "@js-temporal/polyfill";
+import {
+  BASELINE_LOW_TO_HIGH_DURATION,
+  computeBaseline,
+} from "compute-baseline";
+import { Compat, Feature } from "compute-baseline/browser-compat-data";
+import assert from "node:assert/strict";
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { isDeepStrictEqual } from "node:util";
+import YAML, { Document } from "yaml";
+
+/**
+ * Update (or create) a dist YAML file from a feature definition YAML file.
+ *
+ * @param {string} fp The path to the human-authored YAML file.
+ */
+function updateDistFile(fp: string): void {
+  const { dir, name: id } = path.parse(fp);
+  const distPath = path.join(dir, `${id}.dist.yml`);
+
+  const dist = toDist(fp);
+  fs.writeFileSync(distPath, dist.toString({ lineWidth: 0 }));
+}
```

Comment:
Since the header comment is stable, there's no need to read ahead—we can just write out the dist now.

Reply:
Oh. Ignore one of my review comments.